src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -759,6 +805,7 @@ private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, Dat
             scoreColumnName = ctx.Reader.ReadString();
             predictedColumnName = ctx.Reader.ReadString();
             learningRate = ctx.Reader.ReadFloat();
+            useLearningRateScheduling = ctx.Reader.ReadBoolByte();
```

Comment:
>useLearningRateScheduling [](start = 12, length = 25)

why are we serializing this? It is only needed at training and not inferencing, right?

Reply:
We should not be serializing this. This was discussed with Zeeshan and will be addressed by him in his PR.

---
In reply to: [337825750](https://github.com/dotnet/machinelearning/pull/4340#discussion_r337825750) [](ancestors = 337825750)